.github/workflows/integration-tests.yml

```
@@ -72,7 +72,7 @@ jobs:
 
       - name: Install unpackaged python libraries from PyPi
         run: |
-          pip install "tmt[provision]==1.46.0" "tmt[report-junit]==1.46.0" podman pytest pyyaml paramiko
+          pip install "tmt[provision]>=1.32.1" "tmt[report-junit]>=1.32.1" podman pytest pyyaml paramiko
```

NIT: Lets update these
```suggestion
          pip install "tmt[provision]>=1.48.0" "tmt[report-junit]>=1.48.0" podman pytest pyyaml paramiko
```

---

Changed thank you